integration/spark/pom.xml

```diff
@@ -264,6 +269,18 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+      <dependency>
```

Comment:
1. remove redundant dependency
2. scope is test?

Reply:
@QiangCai handled